configuration/openmrs/i18n/registration/locale_fr.json

```diff
@@ -100,6 +100,7 @@
 
   "OBS_ABNORMAL_TRANSLATE_KEY": "Anormal",
   "TRIAGE_INFORMATION": "Triage",
-  "VITAL_SIGNS": "Constantes"
-
+  "VITAL_SIGNS": "signes vitaux",
+  "Enter Phone Number":"Entrer numéro de Téléphone",
```

Comment:
So, should we not replace this by an more 'key-like' label?

Reply:
yes, sure